src/components/lib/Footer.js

```diff
@@ -1,128 +1,66 @@
-
-    return (
-      <footer className={includeSubscription ? "include-subscription" : ""}>
-        <div className="footer-content">
-          <div className="white-paper">
-            <div className="white-paper-label">Learn more about tBTC</div>
-            <div className="white-paper-link">
-              <a
-                href="http://docs.keep.network/tbtc/index.pdf"
-                target="_blank"
-                rel="noopener noreferrer"
-              >
-                Read the Spec {`>>>>`}
-              </a>
-            </div>
-          </div>
-          <a
-            className="download-error-button"
-            onClick={this.handleDownloadError}
-            href={this.state.errorLogUrl}
-            download={`tbtc-dapp-v${version}-log-${new Date().getTime()}.txt`}
-          >
-            Download Error Log ↓
-          </a>
-        </div>
-        {includeSubscription && (
-          <div className="mailing-list">
-            <form onSubmit={this.handleSubmit}>
-              <input
-                type="text"
-                disabled={loading || success}
-                onChange={this.handleInput}
-                value={email}
-                placeholder="enter your email to receive updates"
-              />
-              {success ? (
-                <div className="success">
-                  <Check width="30px" height="30px" />
-                </div>
-              ) : (
-                <input
-                  type="submit"
-                  value={loading ? "(submitting...)" : "Submit >>>>"}
-                />
-              )}
-            </form>
-            {error ? <div className="error">{error}</div> : ""}
-          </div>
-        )}
-        <div className="version-info">{`v${version}`}</div>
-      </footer>
-    )
-  }
-}
-
-Footer.propTypes = {
-  includeSubscription: PropTypes.bool,
+const Footer = () => {
+  return (
+    <footer>
+      <div className="version-info">{`v${version}`}</div>
+      <Tooltip
+        className="help-menu"
+        triggerElement={<button>Help Menu Toggle</button>}
+      >
+        <ul className="help-menu-links">
+          <li>
+            <a
+              href="https://chat.keep.network/"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              Join Our Discord
+            </a>
+          </li>
+          <li>
+            <a
+              href="https://tbtc.network/developers"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              Build with tBTC
+            </a>
+          </li>
+          <li>
+            <a
+              href="https://tbtc.network/developers/how-to-use-the-tbtc-dapp/"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              User Guide
+            </a>
+          </li>
+          <li>
+            <a
+              href="http://docs.keep.network/tbtc/index.pdf"
```

Comment:
Thoughts on pointing this to the HTML version instead (just `tbtc/` instead of `tbtc/index.pdf`)?

Reply:
Ah that's better. Will udpate.

Reply:
Addressed in 9b0c36d